Loop/Managers/Alerts/InAppModalAlertIssuer.swift

```diff
@@ -23,17 +23,13 @@ public class InAppModalAlertIssuer: AlertIssuer {
     typealias TimerFactoryFunction = (TimeInterval, Bool, (() -> Void)?) -> Timer
     private let newTimerFunc: TimerFactoryFunction
 
-    private let soundPlayer: AlertSoundPlayer
-
     init(alertPresenter: AlertPresenter?,
          alertManagerResponder: AlertManagerResponder,
-         soundPlayer: AlertSoundPlayer = DeviceAVSoundPlayer(),
```

Comment:
I think this is good, but I think we need to adjust the `userNotificationCenter` `willPresent` delegate callback to allow notifications to go through while in foreground mode, so we get haptics, sounds, and watch delivery while in the foreground.

Reply:
yes, this is now changed. See `LoopAppManager` line 447